packages/client/modules/newTeam/components/NewTeamForm/NewTeamForm.tsx

```diff
@@ -88,53 +86,33 @@ interface Fields {
 type FieldName = 'orgName' | 'teamName'
 const DEFAULT_FIELD = {value: '', error: undefined, dirty: false}
 
-class NewTeamForm extends Component<Props, State> {
-  state = {
-    isNewOrg: this.props.isNewOrganization,
-    orgId: '',
-    fields: {
-      orgName: {...DEFAULT_FIELD},
-      teamName: {...DEFAULT_FIELD}
-    }
-  }
+const NewTeamForm = (props: Props) => {
+  const {isNewOrganization, organizations} = props
```

Comment:
nit: maybe call this `isInitiallyNewOrg`? something so we don't confuse it with what we're using for state